TESTS/usb_device/msd/mbed_app.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "usb_msd_test",
+    "target_overrides": {
+        "*": {
+            "target.features_add": ["STORAGE"]
+        },
+        "K64F": {
+            "target.components_remove": ["SD"]
```

Comment:
Why does SD need to be removed from the K64F?

Reply:
My assumption was to test embedded flash BD in the first place. In case of `k64f` it has omponents SD and FLASHAPI
https://github.com/ARMmbed/mbed-os/blob/df9ac85cbb38d8f06380b339398e73f968e3ba0a/targets/targets.json#L1443
Because the `BlockDevice::get_default_instance()` implementation uses SD before FLASHAPI
(see code below) we have to remove SD to let `BlockDevice::get_default_instance()` use FLASHAPI
https://github.com/ARMmbed/mbed-os/blob/df9ac85cbb38d8f06380b339398e73f968e3ba0a/features/storage/system_storage/SystemStorage.cpp#L81-L169

Reply:
Perhaps it'd be possible to remove the dependency on the physical device all together? Could you use a HeapBlockDevice instead? https://github.com/ARMmbed/mbed-os/blob/master/features/storage/blockdevice/HeapBlockDevice.h

Reply:
Not really. I'm already using heap block device.
The idea was to test more the one type of block device. The other problem is that many devices has not enough RAM to create 64kB heap block device (it's min size to mount FAT32 fs)

Reply:
> The idea was to test more than one type of block device

I get ya. Though I think the main thing you're trying to test is the USB stack's interaction with the FileSystem API, not the BlockDevice API, correct?

> The other problem is that many devices has not enough RAM to create 64kB heap block device (it's min size to mount FAT32 fs on it)

Dang that's a pain. Crazy idea: maybe there's a way to simulate having a large block device? Especially if you don't actually have to store everything and just need to say the data moving. Almost like a streaming block device or something. (I may be going too far off topic now)